clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClient.java

```diff
@@ -43,36 +45,52 @@
 import io.camunda.zeebe.client.impl.ZeebeClientCloudBuilderImpl;
 import io.camunda.zeebe.client.impl.ZeebeClientImpl;
 
-/** The client to communicate with a Zeebe broker/cluster. */
+/**
+ * @deprecated since 8.6 for removal with 8.8, replaced by {@link io.camunda.client.CamundaClient}
+ *     <p>The client to communicate with a Zeebe broker/cluster.
+ */
+@Deprecated
 public interface ZeebeClient extends AutoCloseable, JobClient {
 
   /**
+   * @deprecated since 8.6 for removal with 8.8, replaced by {@link
+   *     io.camunda.client.CamundaClient#newClient()}
```

Comment:
❓ If the class has the annotation, do the methods still need the annotation? Just wondering because currently only some of the methods in here have the note but not all. That might be confusing for users.